src/types/searchTypes.ts

```diff
@@ -80,20 +80,19 @@ export interface SearchFormAction {
   payload: SearchParams | SearchFilters | string | string[]
 }
 
+export type SearchQueryFilters = [string, string][]
```

Comment:
this is totally a nit, but I think this would be more accurately represented by Record<string, string>.